### PR TITLE
minor improvements

### PR DESCRIPTION
**Pull request by DevriesL**
This PR contains 2 commits:
1, Correct argument name from `button` to `image`.
2, Handle LV_EVENT_LEAVE in gif image object event callback to avoid continuous calling `next_frame_task_cb`.

### CODE REVIEW

File: lv_gif.c [hunk 1]
```diff
@@ -151,3 +154,16 @@ static lv_res_t lv_gif_signal(lv_obj_t * img, lv_signal_t sign, void * param)
...
 }
+
+/**
+ * Event function of the image
+ * @param img pointer to a image object
+ * @param event the triggering event
+ */
+static void lv_gif_event(lv_obj_t * img, lv_event_t event)
+{
+    if (event == LV_EVENT_LEAVE) {
```
Comment (kisvegabor) [inline lv_gif.c:165]: Please add this into `lv_gif_signal` as `LV_SIGNAL_LEAVE`.
If we use an event here the user can't set his own even callback.

**Review by kisvegabor — COMMENTED** [review]
Hi,

Thank you. 
I had only a minor remark.

File: lv_gif.c (hunk 1 above)
Comment (DevriesL) [reply lv_gif.c]: @kisvegabor Thanks for your feedback, I updated the LV_EVENT_LEAVE to LV_SIGNAL_LEAVE, please check.

File: lv_gif.c [hunk 2]
```diff
@@ -119,7 +120,7 @@ static void next_frame_task_cb(lv_task_t * t)
 
     int has_next = gd_get_frame(ext->gif);
     if(has_next == 0) {
-        lv_event_send(img, LV_EVENT_LEAVE, NULL);
```
Comment (kisvegabor) [inline lv_gif.c:122]: `lv_event_send(img, LV_EVENT_LEAVE, NULL);` should be called to let the user know the gif is ended.

Please send the signal first (to let the system process the event) and then send the event (to let the user know about the event).

Comment (DevriesL) [reply lv_gif.c:122]: @kisvegabor Okay, I didn't realize the usage of LV_EVENT_LEAVE, added LV_EVENT_LEAVE back, please check.

**Comment by kisvegabor** [comment]
Perfect, thank you! :slightly_smiling_face:

**Comment by kisvegabor** [comment]
Ups, forgot to merge. Sorry. :sweat_smile:

**Pull request merged**